router/pkg/config/config.schema.json

```diff
@@ -0,0 +1,945 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "version": {
+      "type": "string",
+      "description": "The version of the configuration file. This is used to ensure that the configuration file is compatible.",
+      "enum": ["1"]
+    },
+    "instance_id": {
+      "type": "string",
+      "description": "The unique identifier of the instance. This is used to identify the instance in the control plane and in the metrics."
+    },
+    "graph": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "token": {
+          "type": "string",
+          "description": "The token used to authenticate with other component from Cosmo. Can be ommitted if the router is started with a static execution config."
+        }
+      }
+    },
+    "graphql_metrics": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable the collection of the GraphQL metrics. The default value is true."
+        },
+        "collector_endpoint": {
+          "type": "string",
+          "description": "The endpoint to which the GraphQL metrics are collected. The endpoint is specified as a string with the format 'scheme://host:port'.",
+          "format": "http-url"
+        }
+      }
+    },
+    "dev_mode": {
+      "type": "boolean",
+      "default": false,
+      "description": "Enable the development mode. The development mode is used to enable the development features like ART (Advanced Request Tracing) and pretty logs."
+    },
+    "websocket": {
+      "type": "object",
+      "description": "The configuration for the WebSocket transport. The WebSocket transport is used to enable the WebSocket transport for the GraphQL subscriptions.",
+      "additionalProperties": false,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable WebSocket transport. The default value is true."
+        },
+        "absinthe_protocol": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "default": true,
+              "description": "Enable Absinthe protocol. The default value is true."
+            },
+            "handler_path": {
+              "type": "string",
+              "default": "/absinthe/socket",
+              "description": "The path to the WebSocket handler. The default value is '/absinthe/socket'."
```

Comment:
The path to mount the Absinthe Protocol Handler on. The default value is '/absinthe/socket'.